File: scratch/find_tabs.py
with open('views/profile.php', 'r', encoding='utf-8') as f:
    lines = f.readlines()
for i, line in enumerate(lines):
    if 'nav-tabs' in line or 'tab-content' in line or 'tab-pane' in line:
        print(f"Line {i+1}: {line.strip()}")

File: scratch/find_tabs.php
<?php
$content = file_get_contents(__DIR__ . '/../views/profile.php');
$lines = explode("\n", $content);
foreach ($lines as $index => $line) {
    if (strpos($line, 'nav-tabs') !== false || strpos($line, 'nav nav-') !== false || strpos($line, 'tab-content') !== false) {
        echo "Line " . ($index + 1) . ": " . trim($line) . "\n";
    }
}
?>
